be/src/formats/parquet/group_reader.cpp

```diff
@@ -79,6 +82,18 @@ Status GroupReader::get_next(ChunkPtr* chunk, size_t* row_count) {
     Filter chunk_filter(count, 1);
     DCHECK_EQ(active_chunk->num_rows(), count);
 
+    // row id filter
+    if ((nullptr != _need_skip_rowids) && !_need_skip_rowids->empty()) {
+        std::int64_t current_chunk_base_row = _row_group_first_row + _raw_rows_read - count;
+        auto start_iter = lower_bound(_need_skip_rowids->begin(), _need_skip_rowids->end(), current_chunk_base_row);
+        auto end_iter =
+                upper_bound(_need_skip_rowids->begin(), _need_skip_rowids->end(), current_chunk_base_row + count);
+        for (; start_iter != end_iter; start_iter++) {
+            chunk_filter[*start_iter] = 0;
```

Comment:
I think this code is wrong, chunk_filter size is based on chunk size, its maximum size is 4096, but `*start_iter`'s value is pos in the whole parquet file, you can't do it directly.